lms/djangoapps/discussion_api/api.py

```diff
@@ -274,6 +275,7 @@ def get_thread_list(
         "last_activity_at".
     order_direction: The direction in which to sort the threads by. The only
         values are "asc" or "desc". The default is "desc".
+    discussion_id: ID of an inline discussion for any module
```

Comment:
Can you add a bit more info. about filtering threads based on this id. The doc string says `Return the list of all discussion threads pertaining to the given course` which is not true now.